docs/src/api/Structs.md

```diff
@@ -0,0 +1,8 @@
+# Exported
+
+```@autodocs
+Modules = [PowerSystems]
+Private = false
+Order = [:type]
+Filter = t -> t :> PowerSystems.DynamicInjection
```

Comment:
DynamicInjection looks incorrect here.  Same with `:>`.

Reply:
This file was deleted